test/controllers/amends/CheckYourAnswersControllerSpec.scala

```diff
@@ -71,27 +71,27 @@ class CheckYourAnswersControllerSpec extends SpecBase with SummaryListFluency {
 
         val totalRows: Seq[AmendSummaryRow] = Seq(
           AmendSummaryRow(
-            "Weight of plastic packaging manufactured", "0", None,
+            "Manufactured plastic packaging", "0", None,
```

Comment:
these may clash with my pr, but ive changed them to message keys instead of content, which will fix for both :)